opennebula/resource_opennebula_virtual_router_nic.go

```diff
@@ -209,6 +219,13 @@ func resourceOpennebulaVirtualRouterNICRead(ctx context.Context, d *schema.Resou
 
 	floatingIP, _ := nic.GetStr("FLOATING_IP")
 	floatingOnly, _ := nic.GetStr("FLOATING_ONLY")
+	ip, _ := nic.Get("IP")
+
+	// For VRouter NICs, floating IPs are set using the "IP" field, but it is represented in the ON API
```

Comment:
tiny typo: ONE

Reply:
Hi! I believe this implementation suggestion is incorrect. If you look at implementation of a virtual machine https://registry.terraform.io/providers/OpenNebula/opennebula/latest/docs/resources/virtual_machine#nic there is the `computed_ip` "attribute" at the same time the `ip` is a "parameter". The VR NIC is somewhat similar and I believe it should foloow the same convention. :thinking:

Reply:
Thing is, this is not only computed, this is used to set the IP too. Do we really want two fields, an `ip` to set and a `computed_ip` to get ?